.github/workflows/name: "Python lint and test"  on:   push:     workflow_dispatch:  jobs:   build:     runs-on: "self-hosted"     strategy:       matrix:         python-version: ["3.12"]      steps:       - uses: "actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744" # ratchet:actions/checkout@v3       - name: "Set up Python ${{ matrix.python-version }}"         uses: "actions/python_selfhost.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Python lint and test"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: "self-hosted"
+    strategy:
+      matrix:
+        python-version: ["3.12"]
```

Comment:
Out of curiosity, why a matrix with a single version?

Reply:
Because that's what Bill had in the original that I copied from 😅 I can change it if that's preferred.

Reply:
Because that's what Bill had in the original that I copied from 😅 I can change it if that's preferred.

Reply:
Nah, its fine. Was just curious if you had intentions of building for multiple python versions in the future or if this was maybe overkill.